src/firebase.ts

```diff
@@ -0,0 +1,62 @@
+import { initializeApp } from 'firebase/app'
+import {
+  deleteDoc,
+  doc,
+  DocumentData,
+  getDoc,
+  getFirestore,
+  setDoc
+} from 'firebase/firestore'
+import {
+  ImageMessage,
+  TextMessage,
+  User
+} from './types'
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyCaYDFqWQvrA9nYlg446x0XKHTejrWJea0',
+  authDomain: 'projekty-do-nauki.firebaseapp.com',
+  databaseURL: 'https://projekty-do-nauki-default-rtdb.europe-west1.firebasedatabase.app',
+  projectId: 'projekty-do-nauki',
+  storageBucket: 'projekty-do-nauki.appspot.com',
+  messagingSenderId: '532844117414',
+  appId: '1:532844117414:web:5d766275ce722a24d66367',
+}
+const firebaseapp = initializeApp(firebaseConfig)
+const db = getFirestore(firebaseapp)
```

Comment:
maybe create other file, to instantiate firebase and return access to app and db